tina/config.ts

```diff
@@ -249,6 +226,54 @@ export default defineConfig({
           },
         ],
       },
+      {
+        label: "Machines",
+        name: "machines",
+        path: "content/machines",
+        fields: [
+          {
+            type: "object",
+            label: "Machine Cards",
+            name: "machineCard",
+            list: true, // Indicates that the carousel can have multiple machine cards
+            ui: {
+              itemProps: (item) => {
+                return { label: item?.name };
+              },
+            },
+            fields: [
+              {
+                type: "string",
+                label: "Machine Name",
+                name: "name",
+                required: true,
+              },
+              {
+                type: "image",
+                label: "Machine Image",
+                name: "image",
+                required: true,
+              },
+              {
+                type: "rich-text",
+                label: "Machine Description",
+                name: "description",
+                required: true,
+                description:
+                  "Please select Bullet List in the input field to add Machine's description in the correct format",
+              },
+              {
+                type: "string",
+                label: "Machine Type",
+                name: "type",
+                required: true,
+                description:
+                  "Existing types include Mill, Lathe, Inspection, Engraver, and Bandsaw",
```

Comment:
I left it as a text input field, in case the client wants to add new labels; I'll check with Kayla about that, but I'm leaning to changing that to dropdown if they don't foresee adding new labels.